model/DeliveryFactory.php

```diff
@@ -39,39 +39,50 @@ class DeliveryFactory extends ConfigurableService
 
     /**
      * Creates a new simple delivery
-     * 
+     *
      * @param core_kernel_classes_Class $deliveryClass
      * @param core_kernel_classes_Resource $test
      * @param string $label
      * @return \common_report_Report
      */
     public function create(core_kernel_classes_Class $deliveryClass, core_kernel_classes_Resource $test, $label) {
-        
+
         \common_Logger::i('Creating '.$label.' with '.$test->getLabel().' under '.$deliveryClass->getLabel());
-        
+
         $storage = new TrackedStorage();
-        
+
         $testCompilerClass = \taoTests_models_classes_TestsService::singleton()->getCompilerClass($test);
         $compiler = new $testCompilerClass($test, $storage);
-        
+
         $report = $compiler->compile();
         if ($report->getType() == \common_report_Report::TYPE_SUCCESS) {
             $serviceCall = $report->getData();
-        
+
             $properties = array(
                 RDFS_LABEL => $label,
                 PROPERTY_COMPILEDDELIVERY_DIRECTORY => $storage->getSpawnedDirectoryIds(),
                 DeliveryAssemblyService::PROPERTY_ORIGIN => $test,
             );
 
             foreach ($this->getOption(self::OPTION_PROPERTIES) as $deliveryProperty => $testProperty) {
-                $properties[$deliveryProperty] = $test->getPropertyValues(new \core_kernel_classes_Property($testProperty));
+                $testPropretyInstance = new \core_kernel_classes_Property($testProperty);
+                $validationValue = (string) $testPropretyInstance->getOnePropertyValue(new \core_kernel_classes_Property('http://www.tao.lu/Ontologies/generis.rdf#validationRule'));
```

Comment:
We have constant for this property \oat\tao\helpers\form\ValidationRuleRegistry::PROPERTY_VALIDATION_RULE